src/Footer/Component.tsx

```
@@ -5,9 +5,11 @@ import React from 'react'
 import type { Footer } from '@/payload-types'
 import { Facebook, Instagram, Mail, MapPin, Phone, Twitter, Youtube } from 'lucide-react'
 import TikTok from '@/components/Icons/TikTok'
+import { getLocale } from '@/providers/I18n/server'
 
 export async function Footer() {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Footer'. Consider to delete it or rename it.

'Footer' is defined here:


(lint/suspicious/noRedeclare)